spec.md

```diff
@@ -73,19 +76,22 @@ This specification defines the following terms:
   some action. However, there can also be non-interactive Services that can
   perform the desired actions without direct prompting from the Application.
 
+- *Service Binding*: The representation of an association between an
+  Application and a Service Instance. Service Bindings will often
+  contain the credentials that the Application will use to communicate
+  with the Service Instance.
```

Comment:
I think we need to loosen this a bit since you can request a binding w/o knowing (or ever having) an app to bind to.  Perhaps something like:
```
*Service Binding*: Represents the request to use a Service Instance. As part of
this request there might be a reference to the entity that will use the 
Service Instance. Service Bindings will often contain the credentials that
can then be used to communicate with the Service Instance.
```

Reply:
Yea, I know you just moved it but I think we need to tweak it since we're in this section anyway